src/main/java/rx/schedulers/Schedulers.java

```diff
@@ -137,4 +138,52 @@ public static TestScheduler test() {
     public static Scheduler from(Executor executor) {
         return new ExecutorScheduler(executor);
     }
+    
+    /**
+     * Starts those standard Schedulers which support the SchedulerLifecycle interface.
+     * <p>The operation is idempotent and threadsafe.
+     */
+    public static void start() {
```

Comment:
> Why not make it so shutdown resets it back to nothing running, but if anything is invoked it starts up again just like on a fresh JVM start?

I've started noticing lately that RxComputation threads don't start running until they receive their first task. However, I'm not sure if this behavior is just Java 8u45 or standard. If it isn't standard, replacing the workers with a new set of workers on a shutdown would still leak threads.